kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KubernetesApplicationOperation.scala

```diff
@@ -135,6 +128,7 @@ class KubernetesApplicationOperation extends ApplicationOperation with Logging {
 
 object KubernetesApplicationOperation extends Logging {
   val LABEL_KYUUBI_UNIQUE_KEY = "kyuubi-unique-tag"
+  val SPARK_APP_SELECTOR = "spark-app-selector"
```

Comment:
please change the var name ti align w/ Spark `org.apache.spark.deploy.k8s.Constants`
```
val SPARK_APP_ID_LABEL = "spark-app-selector"
```